spark/SnowplowHelper.scala

```diff
@@ -0,0 +1,30 @@
+package com.liveintent.dwh.fragments
+
+import com.liveintent.dwh.logs.snowplow.models.Snowplow
+import org.apache.spark.rdd.RDD
+
+
+object SnowplowHelper {
+
+  private def removeScrapers[P](data: RDD[P], snowplowExtractor: P => Snowplow): RDD[P] = {
```

Comment:
Since the purpose is to test pure Scala/Spark ability (and not data science), I think it would be helpful to add a comment to this method that explains what it is doing (and what Snowplow is). Knowing that makes it easier to focus on the technical tricks in the code